README.md

```diff
@@ -52,6 +52,10 @@ github "crossroadlabs/Regex"
 
 Run `carthage update` and follow the steps as described in Carthage's [README](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application).
 
+#### Manually
+1. Download and drop ```/Regex``` folder in your project.  
```

Comment:
You would download the ```Zip``` from Github:
<img width="573" alt="screen shot 2016-06-03 at 11 43 18 am" src="https://cloud.githubusercontent.com/assets/6511079/15789328/62d0db46-2980-11e6-9ecf-605b45915c2b.png">